src/book.rb

```diff
@@ -0,0 +1,19 @@
+require_relative 'rental'
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def member?(rental)
+    @rentals.include?(rental)
+  end
+
+  def add_rental(rental)
```

Comment:
- [ ] I will advise you to be explicit about the parameter because it is expected to intake person and date.